src/lib.rs

```diff
@@ -29,6 +29,7 @@ use rand::Rng;
 
 const MAX_CLOCK_SKEW: u64 = 60;
 
+#[derive(Clone, Copy)]
```

Comment:
Err, wait, why are these required? I think `Fernet` should probably be move-only.

Reply:
Cause I didn't want to copy the key and make more identical Fernets myself, ie, the tests.

Reply:
It doesn't seem to be actually used though?

If possible I'd prefer not to have this, it simplifies reasoning about where in memory your keys are.

Reply:
Heh, well, until you through them into a MultiFernet and they jump onto the heap in the Vec. But sure, I can remove that, it just makes the tests a bit more annoying.